### PR TITLE
Readme: Group toolchain list

### PR DESCRIPTION
**Pull request by janpio**
### Checklist
- [x] I've read the [Contribution Guidelines](https://github.com/fastlane/fastlane/blob/master/CONTRIBUTING.md)
- [x] I've updated the documentation if necessary.

### Motivation and Context
As a new user it took a bit of time to wrap my head around all the different tools available in fastlane's awesome toolchain. I was missing a list of what belongs where and is used for what, especially for tools that I don't need right now but could be useful down the lane (*ka-ching).

### Description
I took the list that was already part of the readme and grouped it by OS, then by "subject" of the tools. This way it is a lot easier for newcomers to "classify" all the tools of the fastlane toolchain.

If this is not wanted for the README, I think it should be added somewhere else, possibly the docs.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -127,22 +127,42 @@ If you'd like to take a look at a project already using `fastlane` check out [fa
...
 - [`boarding`](https://github.com/fastlane/boarding): The easiest way to invite your TestFlight beta testers
+
+#### Building
 - [`gym`](https://github.com/fastlane/fastlane/tree/master/gym): Building your iOS apps has never been easier
-- [`match`](https://github.com/fastlane/fastlane/tree/master/match): Easily sync your certificates and profiles across your team using Git
-- [`scan`](https://github.com/fastlane/fastlane/tree/master/scan): The easiest way to run tests for your iOS and Mac apps
-- [`precheck`](https://github.com/fastlane/fastlane/tree/master/precheck): Check your app using a community driven set of App Store review rules to avoid being rejected
+
+#### Plumbing
+- [`spaceship`](https://github.com/fastlane/fastlane/tree/master/spaceship): Ruby library to access the Apple Dev Center and iTunes Connect
```
Comment (ohayon) [inline README.md]: Can we keep this one up with certificates and provisioning profiles for now?

**Review by ohayon — CHANGES_REQUESTED** [review]
Hey @janpio, thanks for this PR! This is a great organizational change. Just one comment about the position of _spaceship_, but otherwise 💯

File: README.md (hunk 1 above)
Comment (janpio) [inline README.md]: Sure - felt "wrong" here anyway. (Although: Spent most time of the PR thinking about a headline for this... :1st_place_medal: )

**Review by KrauseFx — CHANGES_REQUESTED** [review]
.

**Comment by ohayon** [comment]
Hey @janpio, after a bit of discussion with the team, we are going to hold off on merging this change as we are going to be doing some work on our docs in the near future and going to take this type of thing into consideration 😄  Thank you so much for your contribution though and we are really happy to have you in the community! 🚀  🐙  🏎

**Comment by janpio** [comment]
Ok. If I am confused again I can look at the readme at my `patch-1` now ;)

**Comment by KrauseFx** [comment]
Hehe :)

**Comment by janpio** [comment]
Did anything happen in the last month regarding this topic?

**Pull request closed without merging**